### DISCUSSION TITLE
Are Kinematic bodies supposed to trigger collision events?

### DISCUSSION THREAD

**Discussion by edgarssilva** [jcornaz/heron#137] [Q&A] [answered]
I'm using a kinematic player and enemy with corresponding collision layers but they don't trigger collision events. 
But if I make them dynamic they trigger but have physics collision.
Is this supposed behaviour? I would like to have all the collision system without any physics so I can easily do attacks.

**Comment by jcornaz** [comment]
I would expect that they trigger events.

Thouh that's more a question about rapier's behavior than heron (which is only the API).

**Reply by edgarssilva** [reply]
I'll have to check it out then
Edit: Found what I was looking for:  https://rapier.rs/docs/user_guides/bevy_plugin/colliders/#active-collision-types

**Reply by edgarssilva** [reply]
How would you suggest to change the collision types and events? [Rapier Docs](https://rapier.rs/docs/user_guides/rust/colliders#active-collision-types)

**Reply by jcornaz** [reply]
Ah, thanks for the info.

I think heron should enable it by default then.

**Comment by jcornaz** [comment, marked as answer]
Will be solved by #140
